WalletWasabi.Gui/CrashReport/CrashReporter.cs

```diff
@@ -10,36 +10,41 @@ public class CrashReporter
 	{
 		private const int MaxRecursiveCalls = 5;
 		public int Attempts { get; private set; }
-		public string ExceptionString { get; private set; } = null;
+		public string Base64ExceptionString { get; private set; } = null;
 		public bool IsReport { get; private set; }
 		public bool HadException { get; private set; }
 		public bool IsInvokeRequired => !IsReport && HadException;
 		public SerializableException SerializedException { get; private set; }
 
-		public void InvokeCrashReport()
+		public void TryInvokeCrashReport()
 		{
-			var exceptionString = ExceptionString;
-			var prevAttempts = Attempts;
-
-			if (prevAttempts >= MaxRecursiveCalls)
+			try
 			{
-				Logger.LogCritical($"The crash helper has been called {MaxRecursiveCalls} times. Will not continue to avoid recursion errors.");
-				return;
+				if (Attempts >= MaxRecursiveCalls)
+				{
+					throw new InvalidOperationException($"The crash report has been called {MaxRecursiveCalls} times. Will not continue to avoid recursion errors.");
+				}
+				if (string.IsNullOrEmpty(Base64ExceptionString))
+				{
+					throw new InvalidOperationException($"The crash report exception message is empty.");
+				}
+
+				var args = $"crashreport -attempt=\"{Attempts + 1}\" -exception=\"{Base64ExceptionString}\"";
+
+				ProcessBridge processBridge = new ProcessBridge(Process.GetCurrentProcess().MainModule.FileName);
+				processBridge.Start(args, false);
+			}
+			catch (Exception ex)
+			{
+				Logger.LogWarning($"There was a problem while invoking crash report:{ex}.");
```

Comment:
Does `{ex}` behave the same as `{ex.ToString()}` here?

Reply:
https://ideone.com/zpW4Qm

Reply:
Fixed.